### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ericmahare**
### In this project we did the following: -

- [ ] Created classes from the provided UML diagram
- [ ] Created all the methods as described on the project instructions
- [ ] Implemented save functionality to store all user's data on the local storage
- [ ] Added unit tests for all implemented methods.
- [ ] Created Schema for all the classes that we created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,92 @@
...
-[Live Demo Link](https://livedemo.com)
+- Install Ruby in local envinronment.
+- Run program(irb) using the command line
 
+## Prerequisites
+- Understand Ruby basic system and linux
+- No linter errors
 
+## Video link
+- [Video Presentation](https://drive.google.com/file/d/1M8U-6YQBTusXeqdgKs2aOFI9UGJoW1qX/view?usp=sharing)
```
Comment (mahbubul14) [inline README.md]: - [ ] The maximum length of the video should be 5 minutes. Currently, the length of your video is `7.17 minutes.`  Kindly shorten your video and not more than 5 minutes. :camera: 

- [ ] All the members should appear in the video and talk about the project. I can see only two of you in the video. :+1: 

- [ ] You should highlight a piece of code that you find interesting or proud of making. :+1:

File: README.md [hunk 2]
```diff
@@ -1,72 +1,92 @@
...
+   ```
 
-### Setup
+Install the necessary packages
 
-### Install
+```
+  bundle install
+  
+  gem install rubocop
```
Comment (mahbubul14) [inline README.md:40]: - [ ] I couldn't find any `Gemfile` on your project. When I run the command `bundle install`, it also says "Could not locate Gemfile". Please work on it. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @team ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 2 above)
Comment (V0pe) [reply README.md:40]: @mahbubul14 we have added it, Thanks for pointing that out

File: README.md (hunk 1 above)
Comment (V0pe) [reply README.md]: @mahbubul14 We will do it now

**Review by andersonlebon — APPROVED** [review]
## Hi @ericmahare 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**